fetch.bs

```diff
@@ -1417,42 +1417,44 @@ Unless stated otherwise, it is unset.
 "<code>no-cache</code>", "<code>force-cache</code>", or
 "<code>only-if-cached</code>". Unless stated otherwise, it is "<code>default</code>".
 
+<p>Let <var>HTTPCache</var> be the result of determining the <a>HTTP cache</a> partition for
```

Comment:
We've traditionally cased variables in camelCase, so this would be `<var>httpCache</var>`.

Reply:
done